wbooks/src/app/components/ButtonsApp/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { colorsApp } from '@constants/colorsApp';
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  button: {
+    padding: 15,
+    borderRadius: 20,
+    marginTop: 5,
+    marginBottom: 5,
+    alignItems: 'center'
+  },
+  titleButton: {
+    color: colorsApp.white,
+    fontSize: 16,
+    textTransform: 'uppercase'
```

Comment:
We don't typically use that but I think it's fine if the design system specifies it should always be uppercase.
Let's suppose it does.